library/systemd/test/yast2/system_service_test.rb

```diff
@@ -102,6 +102,17 @@
     end
   end
 
+  describe ".build" do
+    let(:systemd_service) { instance_double(Yast::SystemdServiceClass::Service) }
+
+    it "returns a systemd service even if it does not exist" do
```

Comment:
Strictly speaking, `.build` does not return a `SystemdService`. I would write two different statements:

it "returns a service even if the systemd service does not exist"
it "uses a systemd service even if it does not exist"

Reply:
Actually, this method does not know whether the underlying service exists or not. The point is that it calls `build` which always returns a `SystemdService` object.